definitions/npm/knex_v0.12.x/test_knex-v0.12.js

```diff
@@ -92,3 +92,99 @@ knex('foo').havingRaw();
 knex('foo').whereRaw();
 // $ExpectError
 knex('foo').joinRaw();
+
+
+/**
+ * Knex.orWhere()
+ */
+
+knex('foo').orWhere((qb: Knex$QueryBuilder): void => {
```

Comment:
I had to add these 2 annotations otherwise wouldn't know which definition it should use.

```
 * knex_v0.12.x/flow_v0.38.x- (flow-v0.53.1): Unexpected Flow errors(2):
   Error: 0-test_knex-v0.12.js:173
        v-------------------------------
   173: knex('foo').orWhereNot((qb) => {
   174:   qb.where('a', 'b');
   175: });
        -^ call of method `orWhereNot`. Could not decide which case to select
   173: knex('foo').orWhereNot((qb) => {
        ^^^^^^^^^^^^^^^^^^^^^^ function type
     Case 1 may work:
      45:   orWhereNot(builder: Knex$QueryBuilderFn): this,
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ function type. See lib: knex_v0.12.x.js:45
     But if it doesn't, case 4 looks promising too:
      48:   orWhereNot(object: { [string]: any }): this,
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ function type. See lib: knex_v0.12.x.js:48
     Please provide additional annotation(s) to determine whether case 1 works (or consider merging it with case 4):
     173: knex('foo').orWhereNot((qb) => {
                                        ^ return
     173: knex('foo').orWhereNot((qb) => {
                                  ^^ parameter `qb`


   Found 1 error

   null
```